src/main2.cpp

```diff
@@ -459,19 +319,17 @@ int main(int argc, char* argv[])
 	Material grassMaterial(.60f, .001f, .6f, .0001f, grassTextureID, shaderProgram); //just bright, that's all it needs
 	Material plasticMaterial(.5f, .30f, .4f, .1f, plasticTextureID, shaderProgram); //needs to be glossy! This is our racket1
 	Material woodMaterial(.5f, .60f, .5f, .002f, woodTextureID, shaderProgram); //this is you matt
-	Material tattooMaterial(0.5f, 0.2f, 0.5f, 0.002f, tattooTextureID, shaderProgram);
-	Material skinMaterial(.1f, .0f, .66f, .001f, plasticTextureID, shaderProgram); //this is skin
-	Material skyMaterial(.3f, .001f, .9f, .0001f, plasticTextureID, shaderProgram); //Flat blue sky
-	Material bleacherMaterial(.6f, .90f, .6f, .12f, aluminumTextureID, shaderProgram); //Flat blue sky
-
-	//need new way to do bleachers
+	Material skinMaterial(.1f, .0f, .66f, .001f, plasticTextureID, shaderProgram); // This is skin
+	Material skyMaterial(.3f, .001f, .9f, .0001f, plasticTextureID, shaderProgram); // Flat blue sky
+	Material bleacherMaterial(.6f, .90f, .6f, .12f, aluminumTextureID, shaderProgram); // Metal
```

Comment:
we're actually just using metalMaterial for bleachers so this could probably be removed

Reply:
Note: could make bleacherMaterial use the metalTextureID and turn the spec down, or shininess down?

Reply:
Sure, will do